CraftDoc

```diff
@@ -435,7 +435,7 @@ LOOKUP_CACHE_SIZE      = 0
 # normally produced when WARNINGS is set to YES.
 # The default value is: NO.
 
-EXTRACT_ALL            = NO
+EXTRACT_ALL            = YES
```

Comment:
This is the only thing I think we may want to change. My (very limited) understanding is that if we set this to no we'd only be generating doxygen documentation from properly formatted comments. On top of that, if we limit it to the header/source files we've made to isolate our work, the generated documentation would only include stuff that we've worked on.